/**:
  ros__parameters:
    common:
      sensor_height: 2.5 # Sensor height.
      seed_selection_weight: -1.2 # Weight of height that used in initial seed sampling.
    rnr: # Reflected Noise Removal
      min_height_threshold: -0.5 # Minimum height threshold. Initial value = height_noise_margin(=-0.5).
      min_vertical_angle_threshold: -15.0 # Minimum incident angle threshold.
      min_intensity_threshold: 0.2 # Minimum intensity threshold.
    czm: # Concentric Zone Model
      min_range: 2.7 # Minimum radius distance of points to be considered.
      max_range: 100.0 # Maximum radius distance of points to be considered.
      num_zone: 4
      num_near_ring: 4
      min_num_point: 10 # Minimum number of points that should be contained in each zone.
      num_sectors: [16, 24, 24, 24] # Number of sectors for each zone.
      num_rings: [6, 8, 4, 4] # Number of rings of each zone.
      elevation_thresholds: [1.5, 1.5, 1.5, 1.5] # Initial threshold values for elevation.
      flatness_thresholds: [0.0, 0.0, 0.0, 0.0] # Initial threshold values for flatness.
    rpf: # Region-wise Plane Fitting (R-VPF & R-GPF)
      max_vertical_distance_threshold: 0.1 # Maximum distance threshold not to be estimated as vertical points.
      max_angle_threshold: 0.707 # Maximum angle threshold not to be estimated as vertical points.
      max_distance_threshold: 0.125 # Maximum distance threshold not to be estimated as ground points.
      num_iterator: 3 # Number of iteration that Region-wise Plane Fitting is executed.
      num_sample: 20 # Number of initial seed points.
      gpf_seed_margin: 0.3 # Margin of sampling seed for R-GPF
      vpf_seed_margin: 0.25 # Margin of sampling seed for R-VPF
    gle: # Adaptive Ground Likelihood Estimation
      uprightness_threshold: 0.707
      elevation_std_weights: [4.0, 2.0, 2.0, 2.0] # Weights of standard deviation that is used to update elevation for each zone.
      flatness_std_weights: [1.0, 1.0, 1.0, 1.0] # Weights of standard deviation that is used to update flatness for each zone.
      height_noise_margin: -0.5 # Margin that is used to update height noise threshold.
      buffer_storage: 1000
    tgr: # Temporal Ground Revert
      flatness_std_weights: [1.5, 1.5, 1.5, 1.5] # Weights of standard deviation that is used to revert ground for each zone.
